### ISSUE TITLE
Change short form of principles to agree with long form

### ISSUE THREAD

**Issue by bhaugen** [valueflows/valueflows#153]
See http://keimform.de/2016/commons-associations/#comment-1066798

@ChristianSi is right. 

https://valueflo.ws/ dropped this important principle from the long form:

The model must support systems where all the contributors can get shares of the outcome to allocate as they wish e.g.
* contributors to renewable energy coop get kWh of electricity to allocate
* contributors to train system get 'km on the railway' to allocate
* contributors to food network, get 'food baskets' to allocate
* etc.

In other words, a group can choose to introduce various monetary currencies into their flows but can also do all the coordination and accounting without introducing such artifacts.

**Comment by fosterlynn** [comment]
I put the long form of the principles inline with the text, committed directly.
